qlty-coverage/src/ci.rs

```diff
@@ -13,6 +13,8 @@ pub use gitlab::GitLab;
 use qlty_types::tests::v1::CoverageMetadata;
 pub use semaphore::Semaphore;
 
+const QLTY_CI_ACTION_VERSION: &str = "QLTY_CI_ACTION_VERSION";
```

Comment:
This should be UPLOADER_VERSION to align with the names we are using in the protobufs